tests/fsutil/test_fileobj.py

```diff
@@ -282,3 +296,7 @@ def to_protocol(fs: fsspec.AbstractFileSystem):
     if isinstance(fs.protocol, tuple):
         return fs.protocol[0]
     return fs.protocol
+
+
+def to_abs(path: str) -> str:
+    return os.path.abspath(path).replace("\\", "/")
```

Comment:
Comment, rather than change request: this will break if it encounters a file/directory name containing a backslash on a *nix filesystem (where backslashes are allowed). However,

1. This PR isn't the place to change it, since it's just refactoring some inline code.
2. It's very rare to see a backslash in a filename, even if they are legal.
3. It might be argued that anyone foolhardy enough to put backslashes in their filenames deserves what's coming to them :).

Reply:
Thanks for pointing this out. I'll will change that in another PR.